alerting/provider/pushover/pushover.go

```diff
@@ -13,8 +13,9 @@ import (
 )
 
 const (
-	restAPIURL      = "https://api.pushover.net/1/messages.json"
-	defaultPriority = 0
+	restAPIURL                = "https://api.pushover.net/1/messages.json"
+	defaultPriority           = 0
+	defaultPriorityOnResolved = defaultPriority
```

Comment:
Instead of creating a separate constant for defaultPriorityOnResolved that uses defaultPriority anyways, let's just use defaultPriority

Reply:
Done!